### ISSUE TITLE
Escaped characters in HEREDOCs can crash terraform-docs

### ISSUE THREAD

**Issue by jamessthompson** [terraform-docs/terraform-docs#294]
**Describe the bug**

Default values containing backslashed escaped values results in errors such as 

`Error: template: inputs:10:118: executing "inputs" at <.GetValue>: error calling GetValue: json: error calling MarshalJSON for type types.String: invalid character 't' after top-level value`

**To Reproduce**

1. Add the following variable to a project
```
variable "papertrail_log_format" {
  description = "Log format definition - See https://bit.ly/2Qgmv2E"

  default = <<-EOF
  %%{%F %H:%M:%S%z}t
  ip=%h
  cc=%%{client.geo.country_code}V
  city=%%{client.geo.city}V
  ff=%%{if(req.http.Fastly-FF, \"t\", \"f\")}V
  status=%>s
  cache=%%{fastly_info.state}V
  hits=%%{obj.hits}V
  size=%Bb
  time=%%{time.elapsed}Vs
  method=%%{req.request}V
  url=%%{req.url}V
  agent=\"%%{req.http.user-agent}V\"
  ref=%%{req.http.referer}V
  EOF
}
```

2. run `terraform-docs  markdown table .`

3. remove the following two lines from the HEREDOC and step 2 will complete without error
```
  ff=%%{if(req.http.Fastly-FF, \"t\", \"f\")}V
...
  agent=\"%%{req.http.user-agent}V\"
```

**Expected behavior**

It should not parse the contents of the HEREDOC

**Version information:**

- terraform-docs version (use `terraform-docs --version`):
terraform-docs version v0.10.0-alpha darwin/amd64 BuildDate: 2020-07-17T15:35:03-04:00

- Go version (if you manually built. use `go version`):
go version go1.14.5 darwin/amd64

- OS (e.g. Windows, MacOS):
MacOS 10.15.5 (19F101)

**Additional context**

None

**Comment by khos2ow** [comment]
This is quite similar, but not complete duplicate, of #238 and #259 which are fixed by #284. You can give it a go from HEAD of master, or on `v0.10.0-rc.2` which I'll be cutting soon.

**Comment by jamessthompson** [comment]
It does work in HEAD of master.  Thanks!
